### ISSUE TITLE
The accuracy of change dataset packages are not good,Why?

### ISSUE THREAD

**Issue by JiamingZhuo** [jiangboahu/GLCN-tf#6]
I saw the truth that the dataset packages used in the GLCN code are different from the GCN's or GAT's.For instance,GCN use "ind.XX." cora,citeseer,pubmed.
When I  use this relatively standard dataset packages,the performance of GLCN are bad.I get "81.3" in cora.Why  is that? 
Look forward your reply.

**Comment by jiangboahu** [comment]
Thank you for you attention on our work.
The initial version of our code is based on gcn-torch (https://github.com/tkipf/pygcn), and the datasets including its division is provided in gcn-torch. The source data comes from http://www.cs.umd.edu/~sen/lbc-proj/LBC.html. 
The data in the form of "ind.**" are used in the later version of gcn-tensorflow, which is not the same as the original data. For new data, our method may need to adjust some parameters to achieve the desired better performance.

**Comment by JiamingZhuo** [comment]
Thanks for your reply,Jiang.I will test new parameters combination.
There is another thing bothering me . When I deleted "np.random.seed(FLAGS.seed)" and" tf.set_random_seed(FLAGS.seed)",the acc is not good.I believe the random could control the performance of this model,right？Maybe you have other ideas.
Look forward your reply.

**Comment by adityawisnugraha** [comment]
I was trying your code but I got some issues, why the accuracy and loss of validation data are not good?? And I got "nan" in val_loss
![image](https://user-images.githubusercontent.com/18444430/187460070-a3fc687c-ef7e-460d-a5f4-9df198a3f2da.png)
